roles/ceph-common/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: restart ceph mons
+  service: name=ceph-mon-all state=restarted
+  when: socket.rc == 0
+
+- name: restart ceph osds
+  service: name=ceph-osd-all state=restarted
+  when: socket.rc == 0
```

Comment:
The socket will exist if a ceph admin socket is listening. Maybe we could remove it? Thoughts @lyandrew?